zulipterminal/helper.py

```diff
@@ -84,28 +84,25 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
     return wrapper
 
 
-def _set_count_in_model(id_list: List[int], new_count: int,
-                        messages: Dict[int, Message],
+def _set_count_in_model(new_count: int, message_list: List[Message],
                         unread_counts: UnreadCounts) -> None:
     """
         This function doesn't explicitly set counts in model,
         but updates `unread_counts` (which can update the model
         if it's passed in, but is not tied to it).
     """
-    for id in id_list:
-        msg = messages[id]
-
-        if msg['type'] == 'stream':
-            key = (messages[id]['stream_id'], msg['subject'])
```

Comment:
This commit both changes the variable name to `message` and simplifies the function call, so maybe we could be clearer by unifying `messages[id]` and `msg` to `message`, then making the other change in another commit?

Reply:
True. I wanted to change the `msg` name for a while and though this would be a good time :). Will keep the 2 commits separate!